packages/addons/browser/chrome/source/bin/chrome-downloader

```diff
@@ -6,9 +6,18 @@
 . /etc/profile
 oe_setup_addon browser.chrome
 
-ICON=$ADDON_DIR/resources/icon.png
-CONTROL_FILE=/tmp/curl.done
-DATA_FILE=/tmp/curl.data
+ICON="$ADDON_DIR/resources/icon.png"
+CONTROL_FILE="/tmp/curl.done"
+DATA_FILE="/tmp/curl.data"
+CHROME_VERSION_FILE="$ADDON_DIR/config/chrome.version"
+
+[ -f "$CHROME_VERSION_FILE" ] && . "$CHROME_VERSION_FILE"
+if [ -z "$CHROME_VERSION" ]; then
+  kodi-send --action="Notification(No Chrome version specified, Reinstall addon.,30000,${ICON})" >/dev/null
+  exit 0
+fi
```

Comment:
when does this happen ?
would only happen if you fu up at build process or manually change the values ?

Reply:
Yes.

Is maybe a more paranoid check.

Reply:
can you pls drop it, we expect that the addons are okay at the build process, better to not open that pandora's box ;)